docs/modular-resources/examples/tflite-module.md

```diff
@@ -245,8 +245,8 @@ This generated configuration features the minimum required configuration to supp
 
 With everything configured and running, you can now run the inference client that connects to `viam-server` and uses the `example_mlmodelservice_tflite` module.
 
-1. First, determine your robot address and location secret. To do so, navigate to [the Viam app](https://app.viam.com), select the **Code sample** tab, and toggle **Include secret**.
-   The location secret resembles `abcd1234abcd1234abcd1234abcd1234abcd1234abcd1234` and the robot address resembles `my-robot-main.abcdefg123.viam.cloud`.
+1. First, determine your robot address and api key and secret. To do so, navigate to [the Viam app](https://app.viam.com), select the **Code sample** tab, and toggle **Include secret**.
+   The api key secret resembles `abcdef1g23hi45jklm6nopqrstu7vwx8`, the api key id resembles `a1234b5c-678d-9012-3e45-67fabc8d9efa` and the robot address resembles `my-robot-main.abcdefg123.viam.cloud`.
```

Comment:
This is for my own personal clarification: "robot part secret" is something entirely separate from these two values (these two things being what used to be location secret)? And not needed to connect to robot? but they're displayed on the same screen?

Reply:
I think robot part secret is going away and to be replaced with the api key but that's a work in progress which is why they're both there

Reply:
yep! exactly